resources/env/authoring/tomcat-config/crafter/studio/extension/studio-config-override.yaml

```diff
@@ -10,6 +10,8 @@ studio.repo.basePath: ${env:CRAFTER_DATA_DIR}/repos
 # studio.repo.published.live: live
 # Git repository branch for the `staging` publishing target, default "staging"
 # studio.repo.published.staging: staging
+# If Git CLI should be used for some Git operations, for better performance (Git CLI needs to be installed beforehand)
+# studio.repo.git.cli.enabled: true
```

Comment:
Shouldn't this be off by default?

Reply:
By default is off, uncommenting it sets it to true